src/cells/model.ts

```diff
@@ -429,6 +438,15 @@ class CodeCellModel extends BaseCellModel implements ICodeCellModel {
   }
 
   /**
+   * Clear the cell state.
+   */
+  clear(): void {
+    this.output.clear(false);
+    this.executionCount = null;
+    this.input.clear();
```

Comment:
Wow, I didn't think through this name in my suggestion. `input.clear()` sounds like it is clearing the input, not just the prompt. Again, can we just reach in and change the prompt for now, and come up with a better api for the input to be in charge of the prompts later?